charts/drax/values.yaml

```
@@ -1462,7 +1462,16 @@ promtail-notifications:
                   stndDefinedNamespace: event.commonEventHeader.stndDefinedNamespace
                   version: event.commonEventHeader.version
                   timeZoneOffset: event.commonEventHeader.timeZoneOffset
+                  notificationType: event.stndDefinedFields.data.notificationType
                   vesEventListenerVersion: event.commonEventHeader.vesEventListenerVersion
```

This is just nitpicking, but my head couldn't handle the one commonEventHeader below :sweat_smile: 

```suggestion
                  vesEventListenerVersion: event.commonEventHeader.vesEventListenerVersion
                  notificationType: event.stndDefinedFields.data.notificationType
```

---

Maybe something else we want to set in the `json` parse block is `drop_malformed: true`. Then we don't have a potential weird thing going on when we wouldn't have a ves message.